docs/contents/hooks/use-format-byte.ja.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: useFormatByte
+description: "`useFormatByte`はByteを人間が読めるようにフォーマットするためのカスタムフックです。"
+package_name: "@yamada-ui/use-format-byte"
+with_description: true
+label: New
+---
+
+## Import
```

Comment:
Please translate this line into Japanese.

Reply:
Fixed.